sdcm/utils/loader_utils.py

```diff
@@ -175,7 +175,7 @@ def run_prepare_write_cmd(self):
         else:
             self._run_all_stress_cmds(write_queue, params={'stress_cmd': prepare_write_cmd,
                                                            'keyspace_num': keyspace_num,
-                                                           'round_robin': self.params.get('round_robin')})
+                                                           'round_robin': True})
```

Comment:
do we still need this chnage

Reply:
I adapted stress_cmd so we don't need it